scripts/analysis.py

```diff
@@ -1091,32 +1034,229 @@ def plot_power(cursor):
                       'Number of Reactors vs Time',
                       'number_plot', init_year)
 
+
+def plot_in_out_flux(cursor, facility, influx_bool, title, outputname):
+    """plots timeseries outflux from facility name in kg.
```

Comment:
is it outflux or in_out_flux? The name of the function and this docstring disagree.